docs/4.-Reading blockchain-Examples.md

```diff
@@ -42,16 +42,48 @@ Output:
 ### Get one row by index
 
 ```javascript
+// Consider using encodeName, decodeName and bignumber libraries, if you have to do maths on account names.
+const lowerBoundBigNumber = new BigNumber(encodeName('testacc', false));
```

Comment:
These functions only exist in the deprecated, older versions of eosjs. So we wouldn't want those documented here.

Reply:
Fair point.
Maybe we should should pull it back in. I feel we do a library like this. Otherwise we cant use an account_name as foreign_key in a multi_index_table. Others have made the same request:

https://eosio.stackexchange.com/questions/4322/modules-format-encodename-replacement-in-20-0-0

Reply:
I realize its slightly obtuse, but currently if you need the uint64 representation of a name you can use the following:
```                                                                                      
const eosjs = require('eosjs');                                                                                                                             
                                                                                                                                                              
sb = new eosjs.Serialize.SerialBuffer();                                                  
sb.pushName('testacc');                                                                                                                                     
eosjs.Numeric.binaryToDecimal(sb.getUint8Array(8));
```

Reply:
Thanks @jeffreyssmith2nd
Note that you need to pass textEncoder and textDecoder when using nodejs environment
I would add comment with correct example below